src/hooks/useWindowSize.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+// Hook
```

Comment:
this line seems not too useful, maybe instead a jsdoc doc like `/** kasjdf;asjdf. */` ? or just remove it since you have doc below (I would still convert that doc to jsdoc like while you are here)

Reply:
Oh yeah, I totally stole this hook from the internet, I'll remove it. Not sure what you mean by a jsdoc?

Reply:
oh for comments, it's typically awesome if we can use the jsdoc style 

e.g. stolen from https://jsdoc.app/about-getting-started.html :)

```
Use a JSDoc tag to describe your code
/**
 * Represents a book.
 * @constructor
 */
function Book(title, author) {
}
```

Reply:
I'm going to moving all of this to Paragon, so I'll make sure I do that when it's ported.